cookie-banner-rules-list.json

```diff
@@ -6212,6 +6212,15 @@
         "optOut": "[data-cookieman-accept-none]",
         "presence": "#cookieman-modal"
       }
+    },
+    {
+      "id": "43AD2B6B-A57B-4F7A-9D76-E32E696DDC10",
+      "domains": ["dpd.com"],
+      "click": {
+        "optIn": "#popin_tc_privacy_button",
+        "optOut": "#popin_tc_privacy_button_3",
+        "presence": "#popin_tc_privacy_container_button"
```

Comment:
Haven't you thought about targeting something higher up in the tree instead, such as `#tc-privacy-wrapper` or `#popin_tc_privacy`?

Reply:
I don't know why I used #popin_tc_privacy_container_button as I usually use an element that is higher up in the tree. I probably didn't take a good look when I copied the element from the inspector. I can change it, of course. But would there be any advantage at all?

Reply:
Older versions of Firefox (including Firefox 115 ESR) used to hide the element specified in the presence selector before looking for the cookie accept/reject button, so that the user would not have to experience the banner appearing and then disappearing a moment later. Leaving your selector as is would probably only hide the footer with the buttons and leave the rest of the cookie banner visible.

Reply:
The feature was not rolled out before Firefox 120, so that shouldn't be a problem. But I will change it, since it doesn't hurt either. Thanks for the feedback.

Reply:
In fact, by default, only Firefox 120 was affected by this, and Firefox 115 ESR users have to enable this feature at their own risk. But given that it is now out of support, and the cookie banner blocker itself was only released for German users and only works in private windows, this is not a big deal, as you pointed out.

You might as well leave it as is, and no one would probably notice. But on the other hand, Mozilla may still come back to this idea in the future. I personally liked it, but it requires some minor changes to the logic.

Reply:
In fact, by default, only Firefox 120 was affected by this (Firefox 115 ESR users enable this feature at their own risk). But given that it is now out of support, and the cookie banner blocker itself was only released for German users and only works in private windows, this is not a big deal, as you pointed out.

You might as well leave it as is, and no one would probably notice. But on the other hand, Mozilla may still come back to this idea in the future. I personally liked it, but it requires some minor changes to the logic.

Reply:
In fact, by default, only Firefox 120 was affected by this (Firefox 115 ESR users enable this feature at their own risk). But given that it is now out of support, and the cookie banner blocker itself was only rolled out for German users and only works in private windows, this is not a big deal, as you pointed out.

You might as well leave it as is, and no one would probably notice. But on the other hand, Mozilla may still come back to this idea in the future. I personally liked it, but it requires some minor changes to the logic.